### ISSUE TITLE
HTML file is not generated

### ISSUE THREAD

**Issue by sathyaNarayanQA** [KaneCheshire/xcresultviewer#9]
1. Downloaded the latest version 
2.  Followed the read me instructions 
3. When i hit the command its not showing any errors though 

But the html report is not generated in the path.

**Comment by nmazzeo** [comment]
I had same issue as reported above.

**Comment by githubxielijuan** [comment]
I also had the same issue. After running  the command, it did not have any result.

**Comment by KaneCheshire** [comment]
Are you using Xcode 11? It's probably because Xcode 11 outputs xcresult bundles in a different format. I haven't got loads of free time at the moment to look into it but I welcome contributions!

**Comment by jameswrw** [comment]
Same using Xcode 12.3

**Comment by KaneCheshire** [comment]
Yeah because since Xcode 11 Xcode outputs xcresult bundles in a different format to how it was when this repo was created. I've just started a new job so I doubt I'm going to get time to look into this any time soon but as I say, I welcome contributions :)
